src/Network/Network/help/New-AzExpressRouteCircuit.md

```diff
@@ -298,13 +298,13 @@ Accept wildcard characters: False
 ```
 
 ### -SkuTier
-The tier of service for the circuit. Possible values for this parameter are: `Standard`, `Premium` or `Basic`.
+The tier of service for the circuit. Possible values for this parameter are: `Standard`, `Premium` or `Local`.
```

Comment:
so you're replacing this value here but in NewAzureExpressRouteCircuit command you're adding this value, not replacing. can you please confirm?

Reply:
That is correct.